config/initializers/01_redis.rb

```diff
@@ -7,10 +7,3 @@
 REDIS_THROTTLE_POOL = ConnectionPool.new(size: IdentityConfig.store.redis_throttle_pool_size) do
   Redis.new(url: IdentityConfig.store.redis_throttle_url)
 end
-
-REDIS_THROTTLE_ALTERNATE_POOL =
```

Comment:
looks like this constant is still in use in `throttle.rb`?

Reply:
whoops, thank you